src/panels/profile/ha-refresh-tokens-card.ts

```diff
@@ -37,13 +37,25 @@ class HaRefreshTokens extends LitElement {
 
   protected render(): TemplateResult {
     const refreshTokens = this._refreshTokens(this.refreshTokens!);
+    function compareTokenLastUsedAt(tokenA, tokenB) {
```

Comment:
If you want to create a method, move it our of the render function. Also, use a arrow function: `compareTokenLastUsedAt = (tokenA, tokenB) => {...}`